src/app/Fake.Api.Slack/SlackNotification.fs

```diff
@@ -0,0 +1,135 @@
+﻿namespace Fake.Api
+
+#if NETSTANDARD
+open System.Net.Http
+#else
+open System.Net
+#endif
+open Newtonsoft.Json
+
+/// Contains a task to send notification messages to a [Slack](https://slack.com/) webhook
+module Slack = 
+    /// The Slack notification attachment field parameter type
+    [<CLIMutable>]
```

Comment:
I kind of removed `[<CLIMutable>]` when porting records - it had historical reasons, but was not actually used.

Reply:
Got it, removed.